conanfile.py

```diff
@@ -353,6 +368,7 @@ def library_pattern(library):
                               'WebKit']:
                 self.cpp_info.exelinkflags.append('-framework %s' % framework)
             self.cpp_info.sharedlinkflags = self.cpp_info.exelinkflags
+            #self.cpp_info.libs.append('iconv')
```

Comment:
LGTM, after you have removed this line 😄

Reply:
Oops..